src/components/line/details/context.js

```diff
@@ -16,7 +16,8 @@ const Info = ({ title, children }) => (
 )
 
 const Context = () => {
-  const { plugin, context } = useMetadata()
+  const { chartLabels, context } = useMetadata()
+  const plugin = chartLabels?._collect_plugin?.[0]
```

Comment:
Do we need to keep backwards compatibility here?

Reply:
Thanks a lot @novykh . I did research and indeed fallback is needed, i wasn't aware of it. Fixed